app/views/offers/show.html.erb

```diff
@@ -19,25 +25,32 @@
     </div>
   </div>
 
-  <div id="offer-image-preview" class="row">
-    <%= render 'shared/photos_grid_first_two' %>
-  </div>
+  <% if @offer.photos.attached? %>
+    <div id="offer-image-preview" class="row">
+        <%= render 'shared/photos_grid_first_two' %>
+    </div>
+  <% end  %>
 
-  <div id="offer-tags" class="row mt-3">
-    <div class="col">
-        <span class='offer-tag'><%= @offer.price_per_person %>€ pp</span>
-        <span class='offer-tag'><%= @offer.tag.name.capitalize %></span>
-         <span class='offer-tag'>4.0</span>
-        <span class='offer-tag'><%= pluralize(@offer.capacity, 'person') %></span>
-      </div>
-  </div>
 
-  <div id="offer-description" class="row mt-3">
-    <h4>Read more about the experience you go to</h4>
-    <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Incidunt iste commodi reiciendis laboriosam voluptate eius nobis. Reiciendis libero quae quibusdam provident, laudantium officiis optio rem ipsum quia, iusto architecto aliquid!</p>
-  </div>
 
   <div id="offer-reviews" class="col-7 mt-4">
+    <% if @offer.photos.attached? == false %>
```

Comment:
in this case better to check` if @offer.is_external = true`

Reply:
True that, will push this change too in a bit.